include/power/power_state.h

```diff
@@ -0,0 +1,158 @@
+/*
+ * Copyright (c) 2020 Intel corporation
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#ifndef ZEPHYR_INCLUDE_POWER_POWER_STATE_H_
+#define ZEPHYR_INCLUDE_POWER_POWER_STATE_H_
+
+#include <sys/util.h>
+
+#ifdef __cplusplus
+extern "C" {
+#endif
+
+/**
+ * @brief Runtime idle state
+ *
+ * Runtime idle is a system sleep state in which all of the cores
+ * enter deepest possible idle state and wait for interrupts, no
+ * requirements for the devices, leaving them at the states where
+ * they are.
+ */
+#define PM_STATE_RUNTIME_IDLE          BIT(0)
+
+/**
+ * @brief Suspend to idle state
+ *
+ * The system goes through a normal platform suspend where it puts
+ * all of the cores in deepest possible idle state and puts peripherals
```

Comment:
Just to reiterate and clarify my previous comment, this sentence should be changed to "idle state and may put peripherals into low-power states" in order to match state S1 in ACPI, which says "Devices that do not indicate they must remain on **may** be powered off" (from https://en.wikipedia.org/wiki/Advanced_Configuration_and_Power_Interface#Global_states). Powering off the peripherals should be optional.